crates/ide-db/src/symbol_index.rs

```diff
@@ -66,9 +67,14 @@ impl Query {
             mode: SearchMode::Fuzzy,
             assoc_mode: AssocSearchMode::Include,
             case_sensitive: false,
+            include_hidden: false,
```

Comment:
I don't think we have a need for this config here, it is implicitly configured by prefixing `__` to your query